contrib/gitian-descriptors/gitian-linux-parallel.yml

```diff
@@ -0,0 +1,141 @@
+---
+name: "zcash-5.3.0-rc1"
+enable_cache: true
+distro: "debian"
+suites:
+- "buster"
+- "bullseye"
+architectures:
+- "amd64"
+packages:
+- "curl"
+- "autoconf"
+- "automake"
+- "bsdmainutils"
+- "binutils-gold"
+- "ca-certificates"
+- "faketime"
+- "g++-multilib"
+- "git-core"
+- "libc6-dev"
+- "libtinfo5"
+- "libtool"
+- "libxml2"
+- "m4"
+- "ncurses-dev"
+- "pkg-config"
+- "python"
+- "unzip"
+- "wget"
+- "zlib1g-dev"
+remotes:
+- "url": "https://github.com/zcash/zcash.git"
+  "dir": "zcash"
+files: []
+script: |
+  WRAP_DIR=$HOME/wrapped
+  HOSTS="x86_64-linux-gnu"
+  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --enable-hardening --enable-werror"
+  MAKEOPTS="V=1 -j$(nproc)"
+  FAKETIME_HOST_PROGS=""
+  FAKETIME_PROGS="date ar ranlib nm strip objcopy"
+  HOST_CFLAGS=""
+  HOST_CXXFLAGS=""
+  HOST_LDFLAGS=-static-libstdc++
```

Comment:
Why is this needed? I thought it was provided by default.

Reply:
Indeed same as the `gitian-linux.yml`.